lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :scp_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @scp_separator = '/'
+      # %TEMP% == C:\Users\ADMINI~1\AppData\Local\Temp
+      # is a user temp path, not the system path.  Also, it doesn't work, there's
+      # probably an issue with the `ADMINI~1` section
+      @system_temp_path = 'C:\\Windows\\Temp'
+      @external_copy_base = '/programdata'
```

Comment:
@kevpl you can also take the `s` out of the command I gave you to get the non-short version.

```
@external_copy_base = execute('cmd.exe /c "for %I in (%ALLUSERSPROFILE%) do @echo %~I"')
```

This will return `C:\ProgramData` instead of `C:\Progra~3`